### PR TITLE
databases: rewrite headings to follow sentence case Style Guide

### PR DESCRIPTION
**Pull request by ht-l1**
## Because

I am re-writing the headings to follow sentence case per Style Guide.

## This PR

Under databases folder, i made changes to below 4 files:
- databases_and_sql.md
- databases.md
- project_sql_zoo.md
- README.md

## Issue

Related to #26169 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: databases/databases/databases_and_sql.md [hunk 1]
```diff
@@ -139,19 +139,24 @@ If you never quite get to the point where you're comfortable with the really adv
...
+  - [Mucking around with data](#mucking-around-with-data)
+  - [Mashing tables together](#mashing-tables-together)
+  - [Using functions to aggregate your data](#using-functions-to-aggregate-your-data)
+- [SQL is faster than Ruby!](#sql-is-faster-than-ruby)
+- [Assignment](#assignment)
+- [Conclusion](#conclusion)
+- [Knowledge check](#knowledge-check)
+- [Additional resources](#additional-resources)
+
+### Additional resources
```
Comment (ManonLef) [inline databases/databases/databases_and_sql.md:159]:
```suggestion
- [What is the difference between a foreign key and a primary key?](#foreign-key)
- [Where is the setup information for your database stored?](#schema)
- [What are the important parts of a SQL command?](#command-parts)
- [Which SQL statement is associated with "Read" from the CRUD acronym?](#sql-read)
- [Which `JOIN` statement keeps only the rows from both tables where they match up?](#inner-join)
- [How do you use an aggregate function?](#aggregate-function)
- [In which situation would you use the `HAVING` function?](#having-function)
- [Why can't I just use Ruby to process my database data?](#sql-is-faster-than-ruby)

 ### Additional Resources
```

You can click `commit suggestion` if you want to directly commit my proposed reversal, which will avoid you having to locally edit it again and pushing.

**Review by ManonLef — CHANGES_REQUESTED** [review]
Thanks so much for doing this so quickly! 
Something went wrong in one lesson for the `Knowledge check` questions and I've left a comment on it with a fix. If you commit it it can be merged.

**Comment by ht-l1** [comment]
thank you @ManonLef , I just committed your changes. Hope it works now!

**Review by ManonLef — APPROVED** [review]

**Comment by ManonLef** [comment]
Thanks again!

**Pull request merged**